include/pinocchio/multibody/data.hxx

```diff
@@ -10,6 +10,7 @@
 #include "pinocchio/multibody/model.hpp"
 #include "pinocchio/utils/string-generator.hpp"
 #include "pinocchio/multibody/liegroup/liegroup-algo.hpp"
+#include "pinocchio/algorithm/check-data.hpp"
```

Comment:
Usually, this file is included in the `algorithm/xxx.hxx` that will use checkData.

So it's better to patch directly contact-jacobian.hxx and contact-cholesky.hxx.

You can directly include `pinocchio/algorithm/check.hpp` that will include both check-model.hpp and check-data.hpp.